pegasus_en-UK.ts

```diff
@@ -0,0 +1,511 @@
+        <source>Reboot</source>
+        <translation>Reboot</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/MainMenuPanel.qml" line="112"/>
+        <source>Exit Pegasus</source>
+        <translation>Exit Pegasus</translation>
+    </message>
+</context>
+<context>
+    <name>MultifileSelector</name>
+    <message>
+        <location filename="../src/frontend/dialogs/MultifileSelector.qml" line="79"/>
+        <source>This game has multiple entries, which one would you like to launch?</source>
+        <translation>This game has multiple entries, which one would you like to launch?</translation>
+    </message>
+</context>
+<context>
+    <name>NoGamesError</name>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="22"/>
+        <source>No games found :(</source>
+        <translation>No games were found :(</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="23"/>
+        <source>Pegasus couldn&apos;t find any games on your device. If you have not set up Pegasus yet, you can find the documentation here: &lt;i&gt;%1&lt;/i&gt;.&lt;br&gt;If you still see this message, make sure your config files are readable, exist in one of the expected locations and are in the expected format.</source>
+        <translation>Pegasus could not find any games on your device. If you have not set up Pegasus yet, you can find the documentation here: &lt;i&gt;%1&lt;/i&gt;.&lt;br&gt;If you still see this message, make sure your config files are readable, make sure that they are currently located in one of the expected locations and make sure that they are in the expected format.</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="30"/>
+        <source>Please see the log file for more details.</source>
+        <translation>Please see the log file for more details.</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/messages/NoGamesError.qml" line="31"/>
+        <source>You can find it here:&lt;pre&gt;%1&lt;/pre&gt;</source>
+        <translation>You can find it here:&lt;pre&gt;%1&lt;/pre&gt;</translation>
+    </message>
+</context>
+<context>
+    <name>ProviderEditor</name>
+    <message>
+        <location filename="../src/frontend/menu/settings/ProviderEditor.qml" line="83"/>
+        <source>When looking for games, Pegasus can use the following data sources, if enabled:</source>
+        <translation>When looking for games, Pegasus can use the following data sources, if enabled:</translation>
+    </message>
+    <message>
+        <location filename="../src/frontend/menu/settings/ProviderEditor.qml" line="107"/>
+        <source>(nothing on this platform)</source>
+        <translation>(there's no data sources on this platform)</translation>
```

Comment:
This should be *there are*, I think?

Reply:
Both are correct in this case.
But it does sound a bit more proper that way, I will get right on it.